test/test_torch.py

```diff
@@ -3990,6 +4004,32 @@ def test_print(self):
 
             self.assertExpectedInline(str(y), expected_str)
 
+            x = torch.ones(100, 2, 2, 10) * (1 + 1j)
+            y = x.as_strided(size=(100, 2, 10), stride=(2 * 2 * 10, 2 * 10, 1))
+            self.assertEqual(str(y), y.__repr__())
+            expected_str = '''\
+tensor([[[1.+1.j, 1.+1.j, 1.+1.j,  ..., 1.+1.j, 1.+1.j, 1.+1.j],
```

Comment:
@mruberry here's the test for summarize. if a tensor has > 1000 elements, summarize is automatically triggered.

Reply:
Cool - thanks for pointing that out.